company-profiles/clootrack.md

```diff
@@ -0,0 +1,29 @@
+# Clootrack
+
+## Company blurb
+
+Clootrack is a real time customer experience analytics platform that helps brands reduce churn significantly. Clootrack helps brands understand granular qualitative reasons "why" customer experience drops. This reduces response time to CX trends by 88% and increase profitability upto 75%.
```

Comment:
Please remove the %, any statements in this list should be historical and factual without need for references.

Reply:
Updated the statements.